### PR TITLE
UG Updates

### PR DESCRIPTION
**Pull request by yanqiyqh**
Our UG is missing some newly-implemented features. This PR addresses issues #167, #161 and #180

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -262,6 +262,10 @@ Exits the program.
 
 Format: `exit`
 
+### Changing the theme of the app
+
+Click Theme to toggle between Dark Theme and Light Theme.
```
Comment (matthewyip1511) [inline docs/UserGuide.md:267]: Maybe we can add screenshots here to show that dark/light themes look like?

**Review by matthewyip1511 — CHANGES_REQUESTED** [review]
LGTM! Just some small changes

File: docs/UserGuide.md (hunk 1 above)
Comment (zi-yii) [reply docs/UserGuide.md:267]: I agree

Comment (yanqiyqh) [reply docs/UserGuide.md:267]: good idea!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -262,6 +262,14 @@ Exits the program.
...
+#### Dark Theme (default) 
+![Dark Theme (default)](images/darkTheme.png)
```
Comment (matthewyip1511) [inline docs/UserGuide.md:269]: Nice! Thanks for adding screenshots. Merging this PR

**Review by matthewyip1511 — APPROVED** [review]
LGTM! Thanks for the changes 🙂

**Pull request merged**